### DISCUSSION TITLE
Undesired behavior on scrolling once while importing beatmaps. (Constant Scrolling)

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#19490] [Q&A]
Short Video Showcasing the issue: https://streamable.com/zo769q
TLDR: As stated in title beatmaps will continuously scroll when importing beatmaps after scrolling with scroll wheel once until cursor hovers over leaderboard. This is undesired behavior.

**Comment by peppy** [comment]
Move mouse to left to reset scroll. This is intended behaviour - it is keeping your absolute location in the tree because you entered scroll mode.

A future improvement may be to offset absolute positions by newly imported beatmaps, not sure how feasible/easy that will be though.
